myapp/src/components/InputField.js

```diff
@@ -0,0 +1,36 @@
+/**
+ * This is a cutome input field.
+ * This all params are required.
+ *
+ * @param {string} title - use for input labaling
+ * @param {string} value - user given current value
+ * @param {string} type - input tag type
+ * @param {Function} [ e => setfield(e.target.value)] onChange - call back function
```

Comment:
I think we normally indicate `[]` for params when we have optional parameters. But i don't think the `onChange` is optional here

Reply:
Yeah onChange is not a optional params. Fixed it.
aa439fed25ecd2380580da968e5ee9fca0a39698